src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/TriggerAdapterBindingProvider.cs

```diff
@@ -36,6 +35,7 @@ IConverterManager converterManager
         // Listed in precedence for providing via DefaultType.
         // Precedence is more important than how we produce the default type (a direct conversion vs. a converter)
         private static readonly Type[] _defaultTypes = new Type[] {
```

Comment:
I see `_defaultBatchTypes` has similar information. Should that be also updated?

Reply:
I don't think we're supporting batch types for the new ParameterBindingData yet, and we should update `_defaultBatchTypes` when we work on that feature